users/views.py

```diff
@@ -163,7 +163,7 @@ def get(self, request, *args, **kwargs):
             except UserSocialAuth.DoesNotExist:
                 pass
         # clear Django session and get redirect URL
-        response = super().get(request, *args, **kwargs)
+        response = super().dispatch(request, *args, **kwargs)
```

Comment:
Commit message says that this is renamed, but doesn't say why. Would help in the review if the reasoning would be available.

Reply:
@suutari-ai I'll amend the commit. The reason is that django-oidc-provider changed the view to be a base class of LogoutVIew support Django 2.1 (see https://github.com/juanifioren/django-oidc-provider/commit/972071e370cdc0d2798c524e6a4b09681cc727af ) and that resulted in our custom subclass not being called.

Reply:
And that was actually, because in the URL setup, the as_view -method results in dispatch being called. Previously, it was the dispatch of a base view higher up in the class hierarchy, but now it was resolved directly to django-oidc-provider's dispatch which overrided any default dispatch higher up that would have called our get-method.

Reply:
OK, thanks for explaining. Seems very clear now. :+1: